### ISSUE TITLE
Update existing QT Merit list configuration

### ISSUE THREAD

**Issue by nickaddy** [jac-uk/admin#2487]
## Background
The current QT merit list task needs to be reconfigured to display the necessary data on screen, unnecessary content removed, and a download option provided, whilst additional columns not displayed on screen can be optionally selected by Admins to appear on the download.

## User Story
As an Admin, I would like to see on screen the candidate name, the z-score and candidate rank and the additional columns specified below included in the download so that I have all the info necessary for candidate feedback.

## Benefit(s) to user (if not already clear from User story)
On screen info is limited to what is essential for the merit list, additional info is available in the download.

## Feature(s) Description
- [ ] The onscreen merit list display has the following columns: Candidate Full Name, Z-score, Rank and is ordered by Rank ascending, i.e. starting from the highest score ranked number 1
- [ ] Rank is based on overall Z-score as per this example

<img width="362" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBK1RSQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--8055159d9e0fb740578f9c705e54211914e67fed/image.png" alt="image.png" />

- [ ] The QT Merit list download includes the following columns: Application reference, Candidate Full Name, Email, Ethnicity (White/Ethnic minority), Sex, Disability, SJT score, SJT %, CAT score, CAT %, SJT z-score, CAT z-score, Z-score (overall), Rank

## Acceptance Criteria 
It's done when the onscreen and downloaded reports are formatted as described.

## Definition of Done
- [ ] User stories/acceptance criteria met
- [ ] Internal reviews passed (feedback actioned)
- [ ] User testing passed
- [ ] Relevant technical documentation updated
- [ ] User guidance updated
- [ ] Deployed and merged without errors

## User Testing Steps
Specify for users what is being tested (but not how to test it.)

## Feature Champion
Nick

**Comment by nickaddy** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Platform-Development-5ea838cd2aec471eb6d14139/issues/jac-uk/admin/2487?planning-poker) with Zenhub @drieJAC @HalcyonJAC @tomlovesgithub @warrensearle

**Comment by HalcyonJAC** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Platform-Development-5ea838cd2aec471eb6d14139/issues/jac-uk/admin/2487?planning-poker) with Zenhub @KoWeiJAC

**Comment by HalcyonJAC** [comment]
This ticket has been blocked by the work of updating the completed screen of Merit list by @warrensearle.

**Comment by nickaddy** [comment]
@warrensearle I'm planning the output on a live exercise which concludes in January; will get confirmation from the SET and hopefully be able to close the ticket then.

**Comment by nickaddy** [comment]
@warrensearle I'm closing this ticket as this all appears to be in place, however, I'm opening up a new ticket relating to some feedback that Sean has raised.
